state/state.go

```diff
@@ -509,8 +513,11 @@ func (s *State) CloseBatch(ctx context.Context, receipt ProcessingReceipt, dbTx
 
 // isTransactionProcessed determines if the given process transaction response
 // represents a processed transaction.
-func isTransactionProcessed(unprocessedTransaction uint32) bool {
-	return unprocessedTransaction == cFalse
+func isTransactionProcessed(input bool) bool {
+
+	//TODO: Finish this function
+
+	return false
```

Comment:
remove this function, as this logic is actually implemented in `func isUnprocessed(error pb.Error) bool`